src/main/java/io/xpring/xrpl/RippledFlags.java

```diff
@@ -10,9 +10,9 @@
     TF_PARTIAL_PAYMENT(131072);
 
     /** The value of the flag. */
-    public final Integer value;
+    public final int value;
```

Comment:
I just changed these out to be `int`  because thats what the protocol buffers use.